docker/setup/postinstall.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -e
+
+display_error() {
+	echo " !--> ERROR on postinstall:"
+	tail -n 200 /tmp/dockerize.log
+	exit 1
+}
+
+echo " ---> POSTINSTALL"
+
+# Add MySQL-to-Postgres migration script to path (used in entrypoint.sh)
+cp ./docker/mysql-to-postgres/bin/migrate-mysql-to-postgres /usr/local/bin/
+
+# Ensure we can write in /tmp/op_uploaded_files (cf. #29112)
+mkdir -p /tmp/op_uploaded_files/ && chown -R $APP_USER:$APP_USER /tmp/op_uploaded_files/
+
+cp ./packaging/conf/database.yml ./config/database.yml
```

Comment:
@oliverguenther is there any reason to still have the current `config/database.yml` in the repo? I understand mysql support is deprecated anyway?

Reply:
We'll just remove `config/database.yml` so that everything comes from the `DATABASE_URL` env variable.